eidpauth/src/main/java/uk/ac/ucl/eidp/auth/AuthService.java

```diff
@@ -33,7 +35,10 @@ public boolean isAuthorised(String authId, String authToken, Set<String> rolesAl
     UserE user = userController.findByUsernameAndAuthToken(authId, authToken);
     if (user != null) {
       // TODO : Check it might not work
-      return rolesAllowed.containsAll(user.getRoles());
+      Set<String> userRoles = user.getRoles().stream().map(
+              (RoleE role) -> role.getRoleName()
```

Comment:
\[Checkstyle\] WARNING: 'lambda arguments' have incorrect indentation level 14, expected level should be one of the following: 8, 10\.